src/models/index.ts

```diff
@@ -37,7 +37,7 @@ export const DEFAULT_STATE: StateBase = {
   config: DEFAULT_VOYAGER_CONFIG,
   dataset: DEFAULT_DATASET,
   shelf: DEFAULT_SHELF_SPEC,
-  result: DEFAULT_RESULT,
+  result: DEFAULT_RESULT_INDEX,
```

Comment:
I'm not sure why you changed the name to DEFAULT_RESULT_INDEX. It looks like we are using ResultIndex in other places too, but it seems like a confusing name to me.